src/main/java/ca/bc/gov/chefs/etl/forms/ltc/quarterly/processor/LtcQuarterlyYtdApiResponseProcessor.java

```diff
@@ -705,7 +705,7 @@ private List<LtcYtdSubmission> parseYtdQuarterlyRequest(List<Root> ltcQuarterlyY
 			alliedNPRTProfC.setDirCareCostProdHrsOrientationYtd(root.getAlliedNPProdC_item31());
 			alliedNPRTProfC.setDirCareCostProdHrsContractedYtd(root.getAlliedNPProdCCS1());
 			alliedNPRTProfC.setDirCareCostNonProdHrsVacYtd(root.getAlliedNPNProdC_item11());
-			alliedNPRTProfC.setDirCareCostNonProdHrsSickYtd(root.getAlliedNPProdC_item21());
```

Comment:
Did you want to include LTC updates in this branch?

Reply:
I have moved LTC updates to its own branch.